crates/steel-repl/Cargo.toml

```diff
@@ -16,3 +16,4 @@ colored = "2.0.0"
 steel-core = { workspace = true }
 steel-parser = { path = "../steel-parser", version = "0.6.0"}
 dirs = "5.0.1"
+ctrlc = "3.4.4"
```

Comment:
Not directly related to this line change, but I think in this TOML file you'll have to change the workspace version for steel to have the `interrupt` feature flag

Reply:
Hm, I'm not sure what you mean. I did forget to expose the feature at the "top level" package (which I've now changed). I think that should be enough?

Reply:
See this comment: https://github.com/mattwparas/steel/pull/210/files#r1625130105